### PR TITLE
Add user lifecycle reactivate API

### PR DESCRIPTION
**Pull request by mwinckler**

### CODE REVIEW

File: okta/user.go [hunk 1]
```diff
@@ -213,6 +213,23 @@ func (m *UserResource) ActivateUser(userId string, qp *query.Params) (*UserActiv
 	}
 	return userActivationToken, resp, nil
 }
+func (m *UserResource) ReactivateUser(userId string, qp *query.Params) (*UserActivationToken, *Response, error) {
+	url := fmt.Sprintf("/api/v1/users/%v/lifecycle/reactivate", userId)
+	if qp != nil {
+		url = url + qp.String()
```
Comment (quantumew) [inline okta/user.go:219]: Super minor but you can do += here. I am sure it is wrong everywhere else.

Comment (quantumew) [reply okta/user.go:219]: Oh wait you probably grabbed this from the PR you can ignore.

**Review by quantumew — APPROVED** [review]
LGTM!

**Comment by mwinckler** [comment]
@quantumew Please review d8d1aee, adding user credential emails.

**Review by quantumew — APPROVED** [review]
LGTM :shipit:

**Pull request merged**